github/actions_runner_groups.go

```diff
@@ -61,10 +61,27 @@ type SetRunnerGroupRunnersRequest struct {
 	Runners []int64 `json:"runners"`
 }
 
-// ListOrganizationRunnerGroups lists all self-hosted runner groups configured in an organization.
+// ListOrgRunnerGroupOptions extend ListOptions to have the optional parameters VisibleToRepository
+type ListOrgRunnerGroupOptions struct {
+	ListOptions
+
+	// Only return runner groups that are allowed to be used by this repository.
+	VisibleToRepository string `url:"visible_to_repository,omitempty"`
+}
+
+// ListOrganizationRunnerGroups is a wrapper around ListOrganizationRunnerGroupsWithOptions to avoid breaking existing client's code.
+func (s *ActionsService) ListOrganizationRunnerGroups(ctx context.Context, org string, opts *ListOptions) (*RunnerGroups, *Response, error) {
+	listOpts := &ListOrgRunnerGroupOptions{}
+	if opts != nil {
+		listOpts.ListOptions = *opts
+	}
+	return s.ListOrganizationRunnerGroupsWithOptions(ctx, org, listOpts)
+}
+
+// ListOrganizationRunnerGroupsWithOptions lists all self-hosted runner groups configured in an organization.
 //
 // GitHub API docs: https://docs.github.com/en/rest/reference/actions#list-self-hosted-runner-groups-for-an-organization
-func (s *ActionsService) ListOrganizationRunnerGroups(ctx context.Context, org string, opts *ListOptions) (*RunnerGroups, *Response, error) {
+func (s *ActionsService) ListOrganizationRunnerGroupsWithOptions(ctx context.Context, org string, opts *ListOrgRunnerGroupOptions) (*RunnerGroups, *Response, error) {
```

Comment:
Instead of supporting two methods with identical functionality but with an added field,
let's please get rid of this line entirely and go ahead and break the API by just changing line 73 above, as shown.

Reply:
Updated, thanks.